python_modules/dagster/dagster_tests/logging_tests/test_logging.py

```diff
@@ -153,7 +153,7 @@ def test_multiline_logging_complex():
 
     expected_results = [
         (
-            "error_monster - 123 - STEP_FAILURE - DagsterEventType.STEP_FAILURE for step "
+            "error_monster - 123 - start.materialization.output.result.0 - STEP_FAILURE - DagsterEventType.STEP_FAILURE for step "
             "start.materialization.output.result.0"
```

Comment:
This is the only test where the expected output was changed. IMO this is correcting a bug. This is from a test where a step key is defined, but previously the step key wasn't included in the log message, because of sorta convoluted logic around step key in the status quo.